src/libraries/Common/src/Interop/Interop.Ldap.cs

```diff
@@ -96,6 +96,7 @@ internal enum LdapOption
         LDAP_OPT_AREC_EXCLUSIVE = 0x98, // Not Supported in Linux
         LDAP_OPT_SECURITY_CONTEXT = 0x99,
         LDAP_OPT_ROOTDSE_CACHE = 0x9a, // Not Supported in Linux
+        LDAP_OPT_DEBUG_LEVEL = 0x5001,
```

Comment:
Not used? But, not a big deal.

Reply:
Never mind - I hadn't expanded the file.

Reply:
Right, I specifically chose this option as it has been supported in OpenLDAP for a very long time, doesn't require allocation to fetch it, nor a session handle.  As a result it adds very little overhead to our static constructor and should minimally impact existing scenarios (all of which are new for 5.0).